keithasaurus/koda-validate
# Minor Docs fixes in reviewing decorator

## Issue or goal of PR:
Fix minor docs issues (typos/links)
## What I did
fixed 2 typos and a broken link
## How to test
look at diffs, build docs and verify changes. Verify I got the right intended destination for Extensions

- [ ] unit tests were written / updated

## Documentation
Check one:

- [ X] I updated documentation
OR
